code/modules/mining/drilling/distribution.dm

```diff
@@ -178,7 +178,7 @@ Deep minerals:
 								target_turf.resources["silver"] =     rand(RESOURCE_LOW_MIN,RESOURCE_LOW_MAX)
 								target_turf.resources["uranium"] =    rand(RESOURCE_LOW_MIN,RESOURCE_LOW_MAX)
 								target_turf.resources["diamond"] =    0
-								target_turf.resources["phoron"] =     0
```

Comment:
initially when it was added or changing it to plasma? Reason why is it's going off of this line 25 in drill.dm